vite.config.ts

```diff
@@ -0,0 +1,29 @@
+import { resolve } from 'path'
+
+// https://vitejs.dev/config/
+export default {
+  resolve: {
+    alias: [
+      {
+        find: '@',
+        replacement: resolve(__dirname, './src'),
+      },
+    ],
+  },
+  build: {
+    emptyOutDir: false,
```

Comment:
I think you can omit this field (it defaults to true) unless there's a reason you wouldn't want to remove files in the `/dist` folder whenever you're making a build?

Reply:
Ah just kidding, that's so your type files aren't overwritten

Reply:
That's what we're doing in the other projects. I remember we added it for some reason. But I don't remember why. Seems like we shouldn't need it though 🤔 I'll remove.

Reply:
No we for sure need it since the `tsc` command you're chaining puts files in `/dist/types`

Reply:
jk its back! 😂